server/antivirus_connector/cli.py

```diff
@@ -170,11 +170,18 @@ def run_cli(
     rate_limiter: int,
 ):
     if not sequester_service_private_key:
+        sequester_service_private_key = []
         var = os.environ.get("ANTIVIRUS_CONNECTOR_SEQUESTER_SERVICE_PRIVATE_KEY")
```

Comment:
The env variable name isn't descriptive of what it hold (a list of suffix use to find multiple private keys)

```suggestion
        var = os.environ.get("ANTIVIRUS_SEQUESTER_ENV_VAR_SUFFIXES")
```

Reply:
done with another fonctionnal name